public/scripts/suggestion.js

```diff
@@ -0,0 +1,17 @@
+$( document ).ready(function() {
+ $('#git_url').editableSelect();
+ $('#git_url').keyup(() => {
+   let q = $("#git_url").val()
```

Comment:
Also, please use some meaningful variable names, one that define meaning and proper readability.